opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
I have now looked at this, the answer was very simple: it is an infinite recursion! The function calls itself. Simply deleting it will cause the base version to be used. I will fix this and some minor warnings myself to save some time since I have been slow reviewing this. I'm therefore going to push these commits to a new PR and merge from there.